TheFourFifths/consus-client
# Inventory tff235

### Summary

The system to take the quarterly inventory

### Checklist


### How to Review

1.Load it up
2. Go to the new "take inventory" page
3. Notice a list of items you have in the system
4. Scan away! If you put invalid stuff in, you get a toast.
5. If you scan all the items it'll tell you that all items are accounted for.
6. You can, at any point, print the remaining unscanned items.
7. The css is in such away that this page turns into a report when printed.

### Links

- [TFF-235](https://msoese.atlassian.net/browse/TFF-235)
